virtue/virtue-base/sshd_config

```diff
@@ -62,7 +62,6 @@ PasswordAuthentication no
 #GSSAPICleanupCredentials yes
 
 X11Forwarding yes
-X11UseLocalhost no
```

Comment:
What commit did this come from? This wasn't a change made related to resources.

Reply:
I made the change, as it allowed X11 forwarding to work better when I was debugging